packages/protobuf-test/src/gen/js/extra/name-clash_pb.js

```diff
@@ -17,6 +17,19 @@
 /* eslint-disable */
 
 import { proto3 } from "@bufbuild/protobuf";
+import { User } from "./example_pb.js";
+
+/**
+ * This message class will clash with the imported message class
+ *
+ * @generated from message spec.User
+ */
+export const User = proto3.makeMessageType(
```

Comment:
As a result, we end up with two conflicting identifiers - the import, and the declaration.